addons/website/views/snippets/s_image_punchy.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+<template id="s_image_punchy" name="Image Punchy">
+    <section class="s_image_punchy o_cc o_cc2 pt64 pb64">
+        <div class="container">
+            <div class="row o_grid_mode" data-row-count="14">
+                <div class="o_grid_item o_grid_item_image g-height-11 g-col-lg-12 col-lg-12" style="grid-area: 1 / 1 / 12 / 13; z-index: 1">
+                    <img src="/web/image/website.s_image_punchy_default_image" class="figure-img img-fluid rounded" alt=""/>
+                </div>
+                <div class="o_grid_item g-height-6 g-col-lg-7 col-lg-7" style="grid-area: 8 / 6 / 14 / 13; z-index: 2;">
+                    <h2 style="font-size: 7.5rem; text-align: right;">A PUNCHY HEADLINE</h2>
```

Comment:
A bit too "punchy" on mobile 😉

https://github.com/user-attachments/assets/ec426ef5-40fe-47f1-8eef-2489d8f762af

I propose
```suggestion
                    <h2 class="display-1-fs" style="text-align: right;">A PUNCHY HEADLINE</h2>
                    
```

| mobile | desktop |
|--------|--------|
| <img width="848" alt="image" src="https://github.com/user-attachments/assets/742c7d08-33c1-4892-b145-3554961c80d7"> | <img width="1232" alt="image" src="https://github.com/user-attachments/assets/20beecfe-8d57-4424-8cd1-ab911a8f2869"> |

Reply:
Ty for the feedback. I think that if we do that, we loose the "punchy" on desktop. So, what I propose, is to make good use of the editor possibilites and use two headings, one visible on mobile, one visible on desktop, with different sizings in order to keep the punchiness 👊 WDYT ?

Reply:
I'd not do that 😉
The risk is that the user edits the title on desktop without he even noticing that mobile remained unaffected; indeed nothing in the UI it's gonna tell these users that they have to edit mobile as well  😉
This approach you propose it's in the editor possibilities, yes, but it's more for "advanced users" that do that intentionally.
I'd keep the "one block of text" approach, even if the result on mobile it's not great. People will reword this title anyway and they will eventually find their workaround according to the final heading's length 😉

Reply:
If you don't agree  let's discuss that, don't just "resolve" the conversation 😅

Your new value, `font-size: 5.5rem;` ↓

https://github.com/user-attachments/assets/33526157-cb65-4d9f-a20d-565e9794c515

Reply:
> If you don't agree let's discuss that, don't just "resolve" the conversation

Uhm sorry, but yeah I definitely agree. I quick tested it on mobile and it was fine but I guess the resolution was too big, I'll check to make it work on smaller devices too. Was a bit too quick on this one 👍